instrumentation-api-incubator/src/test/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/SqlStatementSanitizerTest.java

```diff
@@ -58,6 +58,15 @@ void veryLongSelectStatementsAreOk() {
     assertThat(result).isEqualTo(expected);
   }
 
+  @ParameterizedTest
+  @ArgumentsSource(DdlArgs.class)
+  void checkDdlOperationStatementsAreOk(
+      String actual, Function<String, SqlStatementInfo> expectFunc) {
+    SqlStatementInfo result = SqlStatementSanitizer.create(true).sanitize(actual);
+    String expected = expectFunc.apply(actual).getFullStatement();
+    assertThat(result.getFullStatement()).isEqualTo(expected);
```

Comment:
This test only verifies that the full statement matches it should also verify that the operation and main identifier match. I have created a PR to add these assertions to `simplifySql` test https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/10108 If you add similar assertions to your test they'll fail.

Reply:
Thanks for pointing out the problem. I've added coverage for `IF | NOT | EXISTS`, and updated tests to correct the assertions.